content/creator/scenes/publishing/publishing.md

```diff
@@ -54,7 +56,11 @@ Open your scene's _scene.json_ file and complete the following data:
 
 ### Via the Decentraland Editor
 
-Make sure you've [installed the Decentraland editor]({{< ref "/content/creator/scenes/getting-started/installation-guide.md#the-decentraland-editor" >}}).
+Make sure you've [installed the Decentraland editor]({{< ref "/content/creator/scenes/getting-started/installation-guide.md#the-decentraland-editor" >}}). 
+
+{{< hint warning >}}
+**📔 Note**:  Deployment to Worlds is not yet supported from the Editor.
```

Comment:
Is this not working already?

Reply:
no, I did a test and it's not possible,

Reply:
I believe it is already working on the editor, yes.
At least there´s a "Publish scene to your world" option in the UI if you click the three dots for more options